src/app.types.tsx

```diff
@@ -196,7 +196,7 @@ export interface TransferToCatalogueItem {
   targetCatalogueCategory: CatalogueCategory | null;
 }
 
-export interface ErrorParsing {
+export interface ApiErrorDetail {
```

Comment:
```suggestion
export interface ApiError {
```
The detail is contained within the response.